README

```diff
@@ -1 +1,12 @@
 https://bteam-dashboard.glob.uno/
+
+For local development, you need Perl with (at least) Mojolicious and
+lib::local installed.  On Ubuntu 18.04, you can install the following packages:
+
+* perl
+* liblocal-lib-perl
+* libmojolicious-perl
```

Comment:
- it's `local::lib` not `lib::local`
- i don't want ubunu specific instructions
- it's better to just have `local::lib` installed at a system level, then use `local::lib` to install other dependencies

ideally this should use a `cpanfile`.  see https://github.com/globau/logbot/blob/master/cpanfile and https://github.com/globau/logbot/blob/master/docs/CONTRIBUTING.md